apps/src/lib/levelbuilder/lesson-editor/FindResourceDialog.jsx

```diff
@@ -56,6 +56,12 @@ class FindResourceDialog extends Component {
             ))}
           </select>
         </label>
+        <p>
```

Comment:
Will the "real" link show in the preview on the right side of the edit page?

Reply:
Great question Dani! I don't think it will be practical to show this during live editing for April, so my thought would be to let the live preview just show the raw `[r ...]` syntax. One idea for the future would be to show these live previews with resolved links after each time the page is saved -- though it seems like we'd have to stop resolving the links as soon as the editor started changing the markdown, unless we want to send ajax requests to resolve them again.

Reply:
I've got a couple ideas of ways we could do various kinds of fanciness to get something like what you're talking about Dave to work, but I agree that they're all too much to try to get done for April.

Reply:
Sounds good! Should we track the follow up in Jira for Q2?

Reply:
Yes, please!